src/agentscope/logging.py

```diff
@@ -6,10 +6,11 @@
 
 from loguru import logger
 
-from .utils.tools import _guess_type_by_extension
```

Comment:
Delete related TextToImageAgent tutorial in [201-agent.md]https://github.com/modelscope/agentscope/blob/main/docs/sphinx_doc/en/source/tutorial/201-agent.md?plain=1#L73